### PR TITLE
[fix] (debug) get_hash_value_fvn DCHECK failed

### LINKED ISSUE #8810: [Bug] get_hash_value_fvn DCHECK failed
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Version

trunk

### What's Wrong?

70b08dd19cc73d83fa9b45ef6719c88c

image

compile failed:
image

### What You Expected?

win

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by xinyiZzz**
# Proposed changes

Issue Number: close #8810

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by cambyzju — COMMENTED** [review]
Should we also need add TypeObject in function RawValue::get_hash_value?

File: be/src/util/runtime_profile.cpp [hunk 1]
```diff
@@ -45,7 +45,7 @@ static const std::string ROOT_COUNTER = "";
 
 RuntimeProfile::RuntimeProfile(const std::string& name, bool is_averaged_profile)
         : _pool(new ObjectPool()),
-          _own_pool(false),
+        //   _own_pool(false),
```
Comment (cambyzju) [inline be/src/util/runtime_profile.cpp:48]: just remove unused codes?

Comment (xinyiZzz) [reply be/src/util/runtime_profile.cpp:48]: https://user-images.githubusercontent.com/13197424/161288577-98d7382e-e476-4638-865a-9929bd5ffe68.png

**Comment by xinyiZzz** [comment]
> Should we also need add TypeObject in function RawValue::get_hash_value?

I understand no, TypeObject is treated as string in other places.

File: be/src/util/runtime_profile.cpp (hunk 1 above)
Comment (xinyiZzz) [reply be/src/util/runtime_profile.cpp]: has been deleted by other PRs

Comment (cambyzju) [reply be/src/util/runtime_profile.cpp]: I means we just remove unused codes instead of comment them.

Comment (xinyiZzz) [reply be/src/util/runtime_profile.cpp]: Because I'm not sure if anyone else will use this later.
It's right to delete it.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**